widgetssdk/src/main/java/com/glia/widgets/di/ControllerFactory.java

```diff
@@ -385,4 +379,15 @@ public EngagementCompletionContract.Controller getEndEngagementController() {
         }
         return engagementCompletionController;
     }
+
+    @NonNull
+    public OperatorRequestHandlerContract.Controller getRequestHandlerController() {
```

Comment:
First association with `getRequestHandlerController()` name is HTTP requests. 🙂 Maybe it makes sense to rename it to `getMediaRequestHandlerController()`?

Reply:
This controller handles all the requests from the operator including screen-sharing requests, what about `getOperatorRequestHandlerController`?

Reply:
Adding my 5 cents )
`getUpgradeRequestsController` ?

Reply:
Is a request for screen-sharing considered an upgrade request?

Reply:
The matter is that `getOperatorRequestHandlerController` is still associating with HTTP requests to me. 🙂 
> Is a request for screen-sharing considered an upgrade request?

Reply:
> Is a request for screen-sharing considered an upgrade request?

Reply:
> Is a request for screen-sharing considered an upgrade request?
Not really, but I like `getUpgradeRequestsController()` or `getEngagementUpgradeController()`.

Reply:
Or `getEngagementUpgradeOffersController()`.

Reply:
Guys, I think in this case we need to change the name for all the components, controller, activitywatcher, and so on.
What do you think regarding these names? Or maybe this will inspire you to suggest a better one.

1. OperatorRequestOrchestratorController
2. OperatorCommandController
3. OperatorRequestManagementController
4. OperatorRequestController

I prefer the 4-th one.
I think an association with an HTTP request is related to the request + handler combination, so this name will reflect the whole functionality.

Reply:
Agree, 4th option is fine to me.